fl4health/losses/deep_mmd_loss.py

```diff
@@ -0,0 +1,224 @@
+from typing import Optional, Tuple
```

Comment:
I think it's worth creating some tests for this loss function before doing any of the refactors I suggest below just to make sure we don't inadvertently break anything in the calculations. It could be as simple as generating a random X, Y and recording the outputs of each function. That way we know we did something if those outputs change with our refactors.

Reply:
I did that and also added a simple test for Deep MMD loss to make sure everything works as expected.